content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -329,6 +329,9 @@ Starting the cluster again will restore it to its previous state.
 The `minikube delete` command can be used to delete your cluster.
 This command shuts down and deletes the Minikube Virtual Machine. No data or state is preserved.
 
+### Upgrading minikube
+See [upgrade minikube](https://minikube.sigs.k8s.io/docs/start/macos/)
```

Comment:
Is that the right hyperlink?

Reply:
I know I'm late by a year and don't know if that got fixed but the link shouldn't include the `/macos/` at the end.

Reply:
https://k8s.io/docs/setup/learning-environment/minikube/ is now a redirect, as it goes.